# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../utf8proc/cross.patch
# Copyright (C) 2007 - 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- utf8proc-v1.1.4/Makefile.vanilla	2009-08-19 19:51:29.000000000 +0200
+++ utf8proc-v1.1.4/Makefile	2009-08-19 19:51:39.000000000 +0200
@@ -5,6 +5,7 @@
 
 cflags = -O2 -std=c99 -pedantic -Wall -fpic $(CFLAGS)
 cc = $(CC) $(cflags)
+ar = ar
 
 
 # meta targets
@@ -29,7 +30,7 @@
 
 libutf8proc.a: utf8proc.o
 	rm -f libutf8proc.a
-	ar rs libutf8proc.a utf8proc.o
+	$(ar) rs libutf8proc.a utf8proc.o
 
 libutf8proc.so: utf8proc.o
 	$(cc) -shared -o libutf8proc.so utf8proc.o
